Corzbank.Helpers/MappingProfile.cs

```diff
@@ -11,7 +11,11 @@ public class MappingProfile: Profile
     {
         public MappingProfile()
         {
-            CreateMap<UserModel, User>();
+            CreateMap<UserModel, User>()
+                .ForMember(u => u.UserName, opt=>opt.MapFrom(x=>string.Join('0', x.Firstname, x.Lastname)));
```

Comment:
opt => opt, always pay attention to formatting. opt.MapFrom( we decided to use lambda as a shortcut to entities, instead of x use us(user model))